app/services/jurisdiction/user_inviter.rb

```diff
@@ -37,8 +37,8 @@ def invite_users
           user.invite!(inviter)
           self.results[:reinvited] << user
         elsif inviter.invitable_roles.include?(user_params[:role].to_s)
-          jurisdiction_id = user_params[:jurisdiction_id] || inviter.jurisdiction&.id
-          user = User.new(user_params.merge({ discarded_at: nil, jurisdiction_id: }))
+          jurisdiction_id = user_params.delete(:jurisdiction_id) || inviter.jurisdictions.pluck(:id)
+          user = User.new(user_params.merge({ discarded_at: nil, jurisdiction_ids: [jurisdiction_id].flatten }))
```

Comment:
wouldn't `inviter.jurisdictions.pluck(:id)` be an array of ids? Will this work if this is the case?

Reply:
that's what the `.flatten` is for

Reply:
@LaurelOlson 
Makes sense, didn't notice the flatten